docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -14,6 +14,39 @@ You can learn about the project setup by checking the [Getting Started](./gettin
 
 :::
 
+## Set up your hardhat project
+
+:::note
+
+For the following guide, we are using `hardhat v2.19.1` and `ethers v6`.
+
+:::
+
+From the folder where you want to create your project, run the following commands and follow the instructions:
+
+```bash
+npx hardhat init
+
+```
+
+### Install the dependencies
+
+Install `dotenv` and `lukso/lsp-smart-contracts` packages:
+
+```bash
+npm install dotenv @lukso/lsp-smart-contracts@0.12.2
+```
+
+### Create a `.env` file
+
+Create a `.env` file in the root of your project and add the following content:
+
+```bash
+PRIVATE_KEY=0x...
+```
+
+Replace the `0x...` with the private key of the account you want to use to deploy your contract or in the case where your are deploying your contract using a Universal Profile, the private key of the EOA that own or has the right permissions on your UP (more information about controllers can be found in the [Key Manager](../../standards/universal-profile/lsp6-key-manager.md) page).
```

Comment:
This is prob duplicated with the getting started page:

https://docs.lukso.tech/learn/smart-contract-developers/getting-started

We put all the setup in this apge so we don't have to duplicate them in the guides.

<img width="1025" alt="image" src="https://github.com/lukso-network/docs/assets/477945/466c199e-f1ad-4be1-a63a-307507a90f5c">


There is already a warning here so i think you can remove it?

Reply:
makes sense I will remove I just want to keep the info about the hardhat version. The problem is that if we add it to the Getting Started page we need to make sure all our guides use the same version. At least `ethers`